docs/blog/2021-07-30-0.4.0-release.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Recoil 0.4
+---
+
+We are pleased to announce the release of Recoil 0.4 with initial Recoil Transactions, selector cache configuration, and other optimizations and fixes.
+
+## Recoil Transactions
```

Comment:
I am dubious that we should brand this as Recoil Transactions™ any more than the other thing should be Recoil Caches™. How about just 

## Updating Multiple Atoms

Reply:
I think the terminology and concept of "transactions" explains this well.  But yeah, the intention wasn't to brand it per-se.  Let me wordsmith it a bit and fix the capitalization, that shouldn't have been capitalized there..